x-pack/solutions/security/plugins/security_solution/public/flyout/entity_details/universal_right/header_data_cards.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useMemo } from 'react';
+import {
+  EuiButtonIcon,
+  EuiCopy,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSuperSelect,
+  EuiTextTruncate,
+} from '@elastic/eui';
+import { css } from '@emotion/react';
+import { i18n } from '@kbn/i18n';
+import { AssetCriticalityBadge } from '../../../entity_analytics/components/asset_criticality';
+import { ResponsiveDataCards } from './components/responsive_data_cards';
+
+const options = [
+  {
+    value: 'unassigned',
+    inputDisplay: (
+      <AssetCriticalityBadge criticalityLevel={'unassigned'} style={{ lineHeight: 'inherit' }} />
+    ),
+  },
+  {
+    value: 'low_impact',
+    inputDisplay: (
+      <AssetCriticalityBadge criticalityLevel={'low_impact'} style={{ lineHeight: 'inherit' }} />
+    ),
+  },
+  {
+    value: 'medium_impact',
+    inputDisplay: (
+      <AssetCriticalityBadge criticalityLevel={'medium_impact'} style={{ lineHeight: 'inherit' }} />
+    ),
+  },
+  {
+    value: 'high_impact',
+    inputDisplay: (
+      <AssetCriticalityBadge criticalityLevel={'high_impact'} style={{ lineHeight: 'inherit' }} />
+    ),
+  },
+  {
+    value: 'extreme_impact',
+    inputDisplay: (
+      <AssetCriticalityBadge
+        criticalityLevel={'extreme_impact'}
+        style={{ lineHeight: 'inherit' }}
+      />
+    ),
+  },
+];
```

Comment:
It's the same implementation of https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/asset_criticality/asset_criticality_selector.tsx, I think we should export options and consume it directly since the implementation is meant to be in sync, and criticality is owned and maintained by entity analytics